contents/product/session-recording.mdx

```diff
@@ -35,6 +35,28 @@ documentation: /docs/user-guides/recordings
 
 <TutorialsSlider topic="session recordings" />
 
+### How does PostHog compare?
+
+|                                       | HotJar | Logrocket |  |   Matomo  |                              PostHog                              |
+|---------------------------------------|:------:|:---------:|:--------:|:---------:|:-----------------------------------------------------------------:|
+|                                                              **Platform**                                                                             |
+| Free plan                             |    ✅   |     ✅     |          |    ✅     |                                 ✅                               |
+| Open source                           |    ❌   |     ❌     |          |    ✅     |                                 ✅                               |
+| Self-hostable                         |    ❌   |     ✅     |          |    ✅     |                                 ✅                               |
+| Cloud hosting                         |    ✅   |     ✅     |          |    ✅     |                                 ✅                               |
+| Product analytics suite               |    ❌   |     ✅     |          |    ✅     |                                 ✅                               |
+|                                                              **Session Recording**                                                                    |
+| Single-page app support               |    ✅   |     ✅     |          |     ✅     |                                 ✅                              |
+| Mobile recordings                     |    ✅   |     ✅     |          |     ✅     |                                 ✅                              |
```

Comment:
I think this line is a little confusing because many people think of mobile apps when they see 'mobile recordings' and I think LogRocket is the only one of these companies that supports that.

Reply:
Updated!